canonical/anbox-cloud.io
# Update dependencies

## Done

- update dependencies
- use latest version of dotrun

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- spot check demo link
